content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,568 @@
+---
+title: 'LoRaWAN® Irrigation System Using Arduino® Edge Control'
+description: "This application note describes how to control a four zones irrigation system using the Edge Control and the Arduino Cloud with LoRaWAN® connectivity"
+difficulty: intermediate
+tags:
+  - Irrigation System
+  - Arduino Edge Control
+  - Arduino MKR WAN 1310
+  - Solenoid Valve
+  - Arduino IoT Cloud
+  - Application Note
+  - WisGate Lite
+  - LoRaWAN®
+author: 'Christopher Mendez'
+libraries:
+  - name: Arduino_EdgeControl
+    url: https://github.com/arduino-libraries/Arduino_EdgeControl
+  - name: ArduinoJson
+    url: https://github.com/bblanchon/ArduinoJson
+  - name: Arduino_ConnectionsHandler
+    url: https://github.com/arduino-libraries/Arduino_ConnectionHandler
+  - name: ArduinoIoTCloud
+    url: https://github.com/arduino-libraries/ArduinoIoTCloud
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+  - arduino-agent
+hardware:
+  - hardware/05.pro-solutions/boards/solutions-and-kits/edge-control
+  - hardware/05.pro-solutions/boards/solutions-and-kits/enclosure-kit
+  - hardware/01.mkr/01.boards/mkr-wan-1310
+---
+
+## Introduction
+
+We know that agricultural activities are normally carried out in remote environments, this makes access to electricity and connectivity a challenge. 
+
+Smart farming techniques are being implemented more and more due to the importance of optimizing the use of resources. This includes the demand for more efficient, eco-friendly, and more profitable crops.
+
+![Application Note Overview. Each pot represents one individual irrigation zone capable of watering a crop field](assets/Thumbnail.png)
+
+Implementing traditional wired communication infrastructure in remote areas can be expensive and time-consuming. LoRaWAN®, being a wireless technology, provides a cost-effective alternative as it requires minimal infrastructure setup, reducing installation and maintenance costs.
+
+Arduino has you covered in these scenarios with its Pro solutions. With products designed to work in remote environments, supplying their power from renewable sources and providing long-distance connectivity and low power consumption.
+
+The shown application note is intended to replicate a scaled smart farming application, that can be implemented on real agriculture fields using the same hardware and firmware.
+
+## Goals
+
+The goal of this application note is to showcase a sensorized farming irrigation system using a combination of an Edge Control, an MKR WAN 1310, and the Arduino IoT Cloud. The project's objectives are the following:
```

Comment:
Last paragraph of the introduction and first paragraph of the goals are really similar, please look for solutions to fusion or differentiate them more.

Reply:
Mention somehow lorawan too